lib/puppet_x/puppetlabs/powershell/powershell_manager.rb

```diff
@@ -41,11 +41,11 @@ def initialize(cmd)
         at_exit { exit }
       end
 
-      def execute(powershell_code, timeout_ms = 300 * 1000)
+      def execute(powershell_code, timeout_ms = nil, working_dir = nil)
```

Comment:
I'm trying to figure out how `timeout_ms` gets set because it seems to me it will always be `nil`, and the only place I see it getting used is in the above invocation  `ps_manager.execute` at powershell.rb:76. I can't seem to find it getting passed in from the resource anywhere via `resource[:timeout]`.  I'm sure I've confused something - is `timeout_ms` something different from the `timeout` resource parameter?